.github/workflows/check-milestone.yml

```
@@ -26,8 +26,8 @@ jobs:
 
             // the github API doesn't expose linked issues, so we have to parse the body ourselves
             function getLinkedIssueId(pull) {
-              // https://regexr.com/7mjnp
-              const match = pull.data.body.match(/(\/issues\/|#)(\d+)/);
+              // https://regexr.com/7sk94
+              const match = pull.data.body.match(/(close(s|d)?|fixe?(s|d)?|resolve(s|d)?) (#|https?:\/\/github.com\/.+\/issues\/)(\d+)/);
```

I believe this would be case-sensitive if we don't add
```suggestion
              const match = pull.data.body.match(/(close(s|d)?|fixe?(s|d)?|resolve(s|d)?) (#|https?:\/\/github.com\/.+\/issues\/)(\d+)/i);
```

I often see it capitalized as `Closes #xxxx` when at the start of issues